common/src/main/java/com/microsoft/identity/common/exception/BrokerCommunicationException.java

```diff
@@ -27,6 +27,8 @@
  * An exception that represents an error where MSAL cannot reach Broker (i.e. through Bind Service or AccountManager).
  */
 public class BrokerCommunicationException extends BaseException {
+    private static final long serialVersionUID = 4959278068787428329L;
+
```

Comment:
Where it is required? And from where this value comes.

Reply:
The compiler generates a warning because this class implements Serializable without an identifier.  I would guess that it's autogenerated by Android Studio based on the class structure.

If you're curious: https://docs.oracle.com/javase/7/docs/api/java/io/Serializable.html

Reply:
Serial version UID in a class is used to ensure that an object of the class is of the particular version of the class.
Yes, this is autogenerated by Android Studio based on the class structure using the plugin GenerateSerialVersionUID